backend/schedulerd/roundrobin_cron.go

```diff
@@ -121,6 +121,7 @@ func (s *RoundRobinCronScheduler) handleEvents(executor *CheckExecutor, ch <-cha
 	}
 }
 
+//nolint
```

Comment:
I think that should be:
```
//nolint:govet
```
It's the `lostcancel` pass of `go vet` that's raising the issue. If there was a way to only disable that specific pass through a comment, I would do that (I don't think there is a way though). Either way, I'd still run all the other linters!